components/organisms/o-header-minimal.vue

```diff
@@ -1,15 +1,17 @@
-<template>
+<template functional>
   <div class="o-header-minimal">
     <ALogo />
   </div>
 </template>
 
 <script>
 import ALogo from 'theme/components/atoms/a-logo';
+import Vue from 'vue'
+
+Vue.component('ALogo', ALogo)
```

Comment:
Is `o-header-minimal` component used in more than 1 place? If no, it might be better to import and register `ALogo` component in the parent. Ofc, not register globally but in `components` section

Reply:
In the future we should use minimal header in checkout as well as an eCommerce best practice.

Reply:
Ok, for now we can register the `ALogo` component in the `Minimal` layout.

Reply:
In the future, we will handle the `ALogo` registration in all the components wherever the minimal header is used.
Also, if we are thinking of registering the SFUI library globally, then I think there will be no need to register the `ALogo` component even in the parent component.